.gitignore

```diff
@@ -16,6 +16,7 @@ release-artifacts/
 
 # Outputs
 coverage.xml
+cover.out
```

Comment:
this is something the repo doesnt create, can you rm this

Reply:
@arkodg the repo creates it [here](https://github.com/envoyproxy/gateway/blob/main/tools/make/kube.mk#L39).

Reply:
ah thanks for highlighting